CONTRIBUTING.md

```diff
@@ -29,26 +29,53 @@ Because we squash all of the changes into a single commit, please try to keep th
 
 For example, `feat(scope): description of feature` should only impact the `scope` package. If your change is a global one, you can use `feat: description of feature`, for example.
 
-### Source code comments
+### Source code comments (NatSpec)
```

Comment:
@d1onys1us I think we should move solidity related guidelines to packages/protocol, as they don't apply to other packages.

Reply:
yes, i will take over your other PR, and then apply them here. but i will do two separate PRs, to minimize the diff for review.

Reply:
My PR is ready.